README.md

```diff
@@ -13,7 +13,8 @@ We **DO NOT** wish the following:
 
 Inspired by many style guides but mainly from [Airbnb](https://github.com/airbnb)'s Style-Guide "style" :-).
 
-**WORK IN PROGRESS. Please be patient.**
+### Why not just read [PEP](https://www.python.org/dev/peps)s?
```

Comment:
We read the whole thing?
We also disagree at times.

maybe change the link to Python's PEP official style guide => https://www.python.org/dev/peps/pep-0008/

You SHOULD read python's PEP 008 as it contains the basic foundations for writing Python code.
This Style Guide aims to share our experience of writing with Python.
It therefore reflects our conclusions and preferences which may contradict with PEP 008.

Reply:
@avraf how this?